dashboard/streamlit_graphs/sales_over_time.py

```diff
@@ -107,23 +125,32 @@ def visualize_sales_per_hour(connection: extensions.connection) -> None:
     Visualizes sales per hour within a date range for the Streamlit dashboard.
     """
     default_end = datetime.now()
-    default_start = default_end - timedelta(days=7)
+    default_start = date(2024, 12, 5)
```

Comment:
Could we make this a magic number if it is referenced in more than one place?

Reply:
i have now added a comment to explain that this was the first day of data collection, and instead of repeating the date value elsewhere, just referenced the variable with that value.